IM_calculation/Advanced_IM/runlibs_2d.py

```diff
@@ -41,6 +41,16 @@ def main(args, im_name, run_script):
     output_dir = args.output_dir
     os.makedirs(output_dir, exist_ok=True)
 
+    station_completed = True
+    for component in ["000", "090"]:
+        component_outdir = os.path.join(output_dir, component)
+        check_status(component_outdir)
+        station_completed = station_completed and check_status(component_outdir)
+    # skip the whole calculation if the station has been ran before
+    if station_completed:
+        print(f"{args} completed")
+        return
+
     model_converged = True
```

Comment:
If it doesn't converge on the first component, we don't need to run the second component